mobify/plugin
# Fixing method return values

Fixing issue where methods invoked on plugin interface don't return the correct value when returning a non-selector based object.

Status: **Ready to merge**

Reviewers: @tedtate

## Changes
- split up the `each` loop inside the plugin to each return the correct return value depending on whether we're initializing the plugin or returning a method value.

## How to test-drive this PR
- checkout this branch
- `grunt test`
- create a plugin that calls a method, and returns a `string`, `number`, `object`, etc.
- ensure that when method is called it returns the correct value
